cem/cem.c

```diff
@@ -4057,24 +4048,27 @@ void InitNormal(void)
     }
 
   // LMV Assign fast and slow weathering portions
-  for (x = 0; x < X_MAX; x++) {
-    for (n = 0; n <= 2 * NUMBER_CHUNK; n++) {
-      for (y = n * CHUNK_LENGTH; y < ((n + 2) * CHUNK_LENGTH); y++) {
-        if (n % 3 == 0 && (!blocks || x > InitialRock - 3)) { /* if even */
-          TypeOfRock[x][y] = 's';
-          topography[x][y] = kCliffHeightSlow;
-        }
-        else { /*if odd */
-          TypeOfRock[x][y] = 'f';
-          topography[x][y] = kCliffHeightFast;
+  {
+    const int chunk_length = Y_MAX / (1 + NUMBER_CHUNK);
+
+    for (x = 0; x < X_MAX; x++) {
+      for (n = 0; n <= 2 * NUMBER_CHUNK; n++) {
+        for (y = n * chunk_length; y < ((n + 2) * chunk_length); y++) {
```

Comment:
I think rather than changing n++ to n+=2, (n+2)  should be (n+1). Otherwise chunk_length really describes half the length of a chunk. But I could be wrong, I'm not too familiar with this yet.